files/prelodr/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "prelodr",
+  "repo": "quintana-dev/prelodr",
+  "files": {
```

Comment:
You need a basePath parameter  set to `dist/` since the files are here https://github.com/quintana-dev/prelodr/tree/1.0.5/dist